controllers/module_reconciler.go

```diff
@@ -374,8 +379,13 @@ func (mrh *moduleReconcilerHelper) handleDevicePlugin(ctx context.Context, mod *
 	}
 
 	logger := log.FromContext(ctx)
+	hashValue, err := hashstructure.Hash(hashData{ModuleVersion: mod.Spec.ModuleLoader.Container.Version}, hashstructure.FormatV2, nil)
```

Comment:
Why don't we hash the kernelVersion here as well?

Reply:
no need, device plugin is independent of kernel version

Reply:
Device plugin is kernel-independent.